src/nrtk_explorer/app/core.py

```diff
@@ -40,7 +40,7 @@
 DEFAULT_DATASETS = [
     f"{DIR_NAME}/coco-od-2017/test_val2017.json",
 ]
-NUM_IMAGES_DEFAULT = 500
+NUM_IMAGES_DEFAULT = 16
```

Comment:
Should we make that configurable via an ENV variable? That way we can override it in deployments.